app/components/user/learning_goal_component.html.erb

```diff
@@ -1,3 +1,3 @@
- <p data-test-id='learning_goal' class="text-gray-500 dark:text-gray-400">
+ <p data-test-id='learning-goal' class="text-gray-500 dark:text-gray-400">
```

Comment:
Standardised these test id values while I was in these files - we we're using a mixture of snake and kebab case.